packages/terminal/src/browser/style/terminal-search.css

```diff
@@ -0,0 +1,105 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.theia-search-terminal-widget-parent {
+    color: var(--theia-ui-font-color1);
+    background: var(--theia-layout-color1);
+
+    position: absolute;
+
+    margin: 0px;
+    border: 1px solid var(--theia-layout-color1);
+    padding: 0px;
+
+    top: 1px;
+    right: 19px;
+
+    z-index: 10;
+}
+
+.search-elem-box {
```

Comment:
all top level css class names should start with `theia-` prefix: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#css-prefix-global-classes